### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MysticExile**
Updates the README.md to be more accurate with the emulator's current progress.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ yuzu emulator
 
 yuzu is an experimental open-source emulator for the Nintendo Switch from the creators of [Citra](https://citra-emu.org/).
 
-It is written in C++ with portability in mind, with builds actively maintained for Windows, Linux and macOS. The emulator is currently only useful for homebrew development and research purposes.
+It is written in C++ with portability in mind, with builds actively maintained for Windows and Linux. The emulator is capable of running several commerical games such as Super Mario Odyssey.
 
-yuzu only emulates a subset of Switch hardware and therefore is generally only useful for running/debugging homebrew applications. yuzu can boot some games, to varying degrees of success.
+yuzu only emulates a subset of Switch hardware and therefore does not run most commercial games at full speed.
```
Comment (Morph1984) [inline README.md]:
```suggestion
yuzu only emulates a subset of Switch hardware and therefore does not run most commercial games at full speed and compatibility.
```

Comment (Morph1984) [reply README.md]: Do you think we should also add a link to the compatibility list? https://yuzu-emu.org/game/

Comment (Morph1984) [reply README.md]: Or, we can have put that in a different sentence

Comment (Morph1984) [reply README.md]: I guess `functionality` would be a better term for that.

Comment (MysticExile) [reply README.md]: Right, I'll add it in.

File: README.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ yuzu emulator
 
 yuzu is an experimental open-source emulator for the Nintendo Switch from the creators of [Citra](https://citra-emu.org/).
 
-It is written in C++ with portability in mind, with builds actively maintained for Windows, Linux and macOS. The emulator is currently only useful for homebrew development and research purposes.
+It is written in C++ with portability in mind, with builds actively maintained for Windows and Linux. The emulator is capable of running several commerical games such as Super Mario Odyssey.
 
-yuzu only emulates a subset of Switch hardware and therefore is generally only useful for running/debugging homebrew applications. yuzu can boot some games, to varying degrees of success.
+yuzu only emulates a subset of Switch hardware and therefore does not run most commercial games at full speed or are most games fully functional.
```
Comment (Morph1984) [inline README.md]:
```suggestion
yuzu only emulates a subset of Switch hardware and therefore most commercial games **do not** run at full speed or are fully functional.
```

File: README.md [hunk 3]
```diff
@@ -6,9 +6,11 @@ yuzu emulator
 
 yuzu is an experimental open-source emulator for the Nintendo Switch from the creators of [Citra](https://citra-emu.org/).
 
-It is written in C++ with portability in mind, with builds actively maintained for Windows, Linux and macOS. The emulator is currently only useful for homebrew development and research purposes.
+It is written in C++ with portability in mind, with builds actively maintained for Windows and Linux. The emulator is capable of running several commerical games such as Super Mario Odyssey.
 
-yuzu only emulates a subset of Switch hardware and therefore is generally only useful for running/debugging homebrew applications. yuzu can boot some games, to varying degrees of success.
+yuzu only emulates a subset of Switch hardware and therefore most commercial games **do not** run at full speed or are fully functional.
+
+Do you want to check which games are compatible and which ones are not? Please visit our [Compatability page](https://yuzu-emu.org/game/)!
```
Comment (Morph1984) [inline README.md]:
```suggestion
Do you want to check which games are compatible and which ones are not? Please visit our [Compatibility page](https://yuzu-emu.org/game/)!
```
nit, fix spelling

Comment (MysticExile) [reply README.md]: Where would I be without you <3

**Pull request merged**